pkg/protocol/engine/booker/inmemorybooker/booker.go

```diff
@@ -116,6 +116,9 @@ func (b *Booker) Queue(block *blocks.Block) error {
 	return nil
 }
 
+// Reset resets the component to a clean state as if it was created at the last commitment.
+func (b *Booker) Reset() {}
```

Comment:
Missing implementation? If nothing to reset, then add it in a comment please.

Reply:
I added a comment everywhere, where its empty just to comply with an upcoming interface.